platform/android/audio_driver_jandroid.cpp

```diff
@@ -79,8 +79,7 @@ Error AudioDriverAndroid::init() {
 	JNIEnv *env = ThreadAndroid::get_env();
 	int mix_rate = GLOBAL_DEF("audio/mix_rate", 44100);
 
-	int latency = GLOBAL_DEF("audio/output_latency", 25);
-	latency = 50;
+	int latency = GLOBAL_DEF("audio/output_latency", 50);
```

Comment:
It seems like I'm introducing a bug here instead.  I am seeing `GLOBAL_DEF("audio/output_latency", 25)` way too many times everywhere else:

```
servers/audio/audio_driver_dummy.cpp:   int latency = GLOBAL_DEF("audio/output_latency", 25);
drivers/alsa/audio_driver_alsa.cpp:     int latency = GLOBAL_DEF("audio/output_latency", 25);
drivers/xaudio2/audio_driver_xaudio2.cpp:       int latency = GLOBAL_DEF("audio/output_latency", 25);
drivers/pulseaudio/audio_driver_pulseaudio.cpp: int latency = GLOBAL_DEF("audio/output_latency", 25);
drivers/rtaudio/audio_driver_rtaudio.cpp:       int latency = GLOBAL_DEF("audio/output_latency", 25);
platform/osx/audio_driver_osx.cpp:      int latency = GLOBAL_DEF("audio/output_latency", 25);
platform/haiku/audio_driver_media_kit.cpp:      int latency = GLOBAL_DEF("audio/output_latency", 25);
```

I think it's a safe bet to let the `25` live.